packages/frontend/public/index.html

```diff
@@ -7,9 +7,26 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Acknowledge community contributions, build a culture of giving and gratitude."
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="%PUBLIC_URL%/apple-touch-icon.png"
```

Comment:
Should we add `PUBLIC_URL=` to `.env.template` to make sure we don't forget to define the public url?

Reply:
I believe `PUBLIC_URL` should only be defined when app is not hosted at the web server root.

Reply:
i see, makes sense